resources/com/mesosphere/global/terraform_file_deploy.sh

```diff
@@ -177,6 +177,99 @@ EOF
   else
     echo -e "\e[32m nginx reached \e[0m"
   fi
+  echo -e "\e[34m deploying dotnet-sample \e[0m"
+  dcos marathon app add <<EOF
+{
+  "labels": {
+    "HAPROXY_DEPLOYMENT_GROUP": "dotnet-sample",
+    "HAPROXY_0_REDIRECT_TO_HTTPS": "true",
+    "HAPROXY_GROUP": "external",
+    "HAPROXY_DEPLOYMENT_ALT_PORT": "10005",
+    "HAPROXY_0_VHOST": $(terraform output public-agents-loadbalancer)
+  },
+  "id": "/dotnet-sample",
+  "acceptedResourceRoles": [
+    "*"
+  ],
+  "constraints": [
+    [
+      "os",
+      "LIKE",
+      "windows"
+    ]
+  ],
+  "container": {
+    "portMappings": [
+      {
+        "containerPort": 80,
+        "hostPort": 33333,
+        "protocol": "tcp",
+        "servicePort": 10004,
+        "name": "http"
+      }
+    ],
+    "type": "DOCKER",
+    "volumes": [],
+    "docker": {
+      "image": "mcr.microsoft.com/dotnet/core/samples:aspnetapp",
```

Comment:
that image also runs on linux machines i see the constraint but isn't there something which only runs on windows?

Reply:
Intention to start the .NET Core application on Windows platform only

Reply:
we must change it, so it will not run for all integration tests at the end

Reply:
@sebbrandt87 , got it, I'm going to add flag --no-windows to exclude windows test

Reply:
maybe --windows would be better ;), only include the windows specific definitions if that part is set

Reply:
fixed at 190bc784c1ed8f11a9b48975c603d05a598485df